.github/workflows/ci.yml

```diff
@@ -0,0 +1,502 @@
+            search-engine-type: elasticsearch
+            db-type: postgresql
+          - testSuite: Elasticsearch Basic Auth MySQL
+            profiles: all-tas-tests,elastic-basic-auth
+            search-engine-type: elasticsearch
+            db-type: mysql
+          - testSuite: Elasticsearch Basic Auth MariaDB 10.6
+            profiles: all-tas-tests,elastic-basic-auth
+            search-engine-type: elasticsearch
+            db-type: mariadb
+          - testSuite: Elasticsearch Basic Auth MS SQL
+            profiles: all-tas-tests,elastic-basic-auth
+            search-engine-type: elasticsearch
+            db-type: mssql
+          - testSuite: Elasticsearch Basic Auth Oracle 19.3.0-ee
+            profiles: all-tas-tests,elastic-basic-auth
+            search-engine-type: elasticsearch
+            db-type: oracle
+          - testSuite: Opensearch Basic Auth
+            profiles: all-tas-tests,elastic-basic-auth
+            search-engine-type: opensearch
+            db-type: postgresql
+    steps:
+      - uses: actions/checkout@v3
+      - uses: Alfresco/alfresco-build-tools/.github/actions/get-build-info@v1.30.0
+      - uses: Alfresco/alfresco-build-tools/.github/actions/setup-java-build@v1.30.0
+      - name: "Build"
+        timeout-minutes: ${{ fromJSON(env.GITHUB_ACTIONS_DEPLOY_TIMEOUT) }}
+        run: |
+          bash ./scripts/ci/init.sh
+          bash ./scripts/ci/build.sh
+      - name: "Run tests"
+        timeout-minutes: 30
+        run: mvn -B install -ntp -pl ":content-repository-elasticsearch-test" -am -P${{ matrix.profiles }} -Denvironment=default -DrunBugs=false "-Dsearch.engine.type=${{ matrix.search-engine-type }}" "-Ddatabase.type=${{ matrix.db-type }}"
+      - name: "Clean Maven cache"
+        run: bash ./scripts/ci/cleanup_cache.sh
+
+  upgrade_tas_tests:
+    name: ${{ matrix.testSuite }} Upgrade TAS tests
+    runs-on: ubuntu-latest
+    if: >
+      !contains(github.event.head_commit.message, '[skip search]') &&
+      !contains(github.event.head_commit.message, '[skip tests]')
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - testSuite: Elasticsearch
+            search-engine-type: elasticsearch
+          - testSuite: Opensearch
+            search-engine-type: opensearch
+    steps:
+      - uses: actions/checkout@v3
+      - uses: Alfresco/alfresco-build-tools/.github/actions/get-build-info@v1.30.0
+      - uses: Alfresco/alfresco-build-tools/.github/actions/setup-java-build@v1.30.0
+      - name: "Build"
+        timeout-minutes: ${{ fromJSON(env.GITHUB_ACTIONS_DEPLOY_TIMEOUT) }}
+        run: |
+          bash ./scripts/ci/init.sh
+          bash ./scripts/ci/build.sh
+      - name: "Configure AWS credentials"
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: eu-west-1
```

Comment:
Done. We need to set it everytime in the action because otherwise it causes the "Input required and not supplied: aws-region" [error](https://github.com/Alfresco/acs-packaging/actions/runs/3902012718/jobs/6664561922#step:6:37).